.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[{package}.json]
```

Comment:
@nlok5923 , why do we need to spcify this again on [{package}.json] file type? won't line 3-5 above cover this type of file as well (with [*])?

Reply:
yes @mtreacy002  you were right will do that changes in the latest commit thanks.